src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerCreateHearingSummary.java

```diff
@@ -121,7 +128,11 @@ public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(
         caseData.setListing(recordListHelper.saveSummary(details.getData()));
         caseData.setCurrentEvent("");
 
-        updateCategoryToCaseworkerDocument(caseData.getListing().getSummary().getRecFile());
+        List<ListValue<CaseworkerCICDocumentUpload>> uploadedDocuments = caseData.getListing().getSummary().getRecFileUpload();
```

Comment:
can we move this repeated code block into a common method and make use of it in all other files?

Reply:
The only repeated lines across all events are:
List<ListValue<CaseworkerCICDocument>> documents = convertToCaseworkerCICDocumentUpload(uploadedDocuments, false);
        updateCategoryToCaseworkerDocument(documents);
      
I can convert those so it is only one line. Also I can create a common method for Create Hearing Summary and Edit Hearing Summary to use since that is repeated code. Would this be acceptable?

Reply:
Look for possibilities to move the below common code where it is repeated in few files, along with the ones you highlighted above.
```
caseData.getListing().getSummary().setRecFile(documents);
caseData.getListing().getSummary().setRecFileUpload(new ArrayList<>());
```

yep, am ok to create the common method.